models/api_rate.go

```diff
@@ -65,10 +65,6 @@ func (r *Rate) SpecialServicesRequested() *SpecialServicesRequested {
 func (r *Rate) Weight() Weight {
 	commoditiesWeight := r.Commodities.Weight()
 	if !commoditiesWeight.IsZero() {
-		// Assume the weight must be between than 13 and 150 lbs.
-		// If the weight is less than 13 lbs, assume a weight of 13 lbs, which is
-		// heavy enough that the destination will matter when choosing between two
-		// fedex ground rates
```

Comment:
Since the 150 constraint is not removed, probably we don't need fully remove the comment but just need some modifications

Reply:
Comments are not a good documentation practice, as they do not change any behavior, as in this PR for example, whether merging or not the code continues to do the same.

Furthermore, when the code changes if we don't update the comment it describes false behavior.

Reply:
@llacerdapaypal I am not saying we need keep the original comment, but you don't necessarily need delete all these lines without adding new comment. The context that we make assumption of weight less than 150lb is missing then.

Reply:
I tried to be more explicit with variables and functions about the units of measurement, the maximum value, and the empty weight safeguard instead of using comments. What do you think?